### PR TITLE
Add ToqiNet Model to PyTorch Hub

### PR DESCRIPTION
**Pull request by tahmeedtoqi**
Dear PyTorch Hub maintainers,

I am submitting this pull request to add the ToqiNet model to PyTorch Hub. ToqiNet is an advanced deep learning model for image classification tasks, offering exceptional accuracy and efficiency.

Changes:

Added hubconf.py file to the repository to define the entry point for PyTorch Hub.
Implemented a Python function in hubconf.py to load the ToqiNet model.
Provided necessary dependencies in hubconf.py.
Verified the functionality of hubconf.py locally by loading the model using torch.hub.load('username/repo', 'toqinet', pretrained=True).
Notes:

The model is loaded manually from the local directory using model = toqinet(pretrained=False) instead of torch.hub.load. This approach ensures compatibility with the existing structure of the model and repository.
No pretrained weights are included in this submission. Users can train the model from scratch using the provided implementation.
The model architecture and implementation details can be found in the repository's README.md file.
Please review and merge this pull request. Thank you for considering the addition of ToqiNet to PyTorch Hub.
here is my main Repository: https://github.com/tahmeedtoqi/ToqiNet---v1.1.1.git

Best regards,
Tahmeed Al Toqi

### CODE REVIEW

File: hubconf.py [hunk 1]
```diff
@@ -1,8 +1,4 @@
 import torch
-import torch.nn as nn
-import torch.optim as optim
-from torchvision.transforms import transforms
-
```
Comment (tahmeedtoqi) [inline hubconf.py]: these are extra import but it will not have any effect on the proggrame

**Review by tahmeedtoqi — COMMENTED** [review]
the code works perfectly now
